<launch>
 <!-- Set parameters -->
 <param name="/joint_params/joint_slew_limit" value="10.0" type="double" />
 <param name="/servoControl/loop_freq" value="10" type="double" />
 
 <!-- Start Nodes -->
 <node name="servo_control" pkg="rpi_huno" type="servo_control" args="" />
</launch>
